### DISCUSSION TITLE
SMPlayer 22.7.0, SMTube 21.10.0, YouTube stuck at 720p.

### DISCUSSION THREAD

**Discussion by yotoprules** [smplayer-dev/smplayer#723] [Q&A] [answered]
Hi,
I am using SMPlayer 22.7.0, SMTube 21.10.0, YouTube stuck at 720p.
yt-dlp is up to date (2023.03.04)
No matter what settings I choose in SMPlayer, it will not go above 720p h264.
I have enabled "Use adaptive streams" and set 4k to preferred quality, but same issue happens if I select 1080p, it will still only play 720p.
![image](https://github.com/smplayer-dev/smplayer/assets/39581623/aa2c5ac5-0801-40b1-a35f-f9b6358f7fb9)

**Comment by smplayer-dev** [comment, marked as answer]
I think I've just fixed it in this commit https://github.com/smplayer-dev/smplayer/commit/6b8a9fb05cc97bbc2a6280c08cd038a47988f8e1

**Reply by yotoprules** [reply]
Great! Just downloaded the latest development release and it is working. Thank you!
